tests/FakeItEasy.Analyzer.Tests/UnusedCallSpecificationAnalyzerTests.cs

```diff
@@ -97,17 +98,115 @@ interface IFoo { int Bar(); }
                 test,
                 new DiagnosticResult
                 {
-                    Id = UnusedCallSpecificationAnalyzer.DiagnosticId,
+                    Id = DiagnosticDefinitions.UnusedCallSpecification.Id,
                     Message =
-                        "Unused call specification for 'foo.Bar()'; did you forget to configure or assert the call?",
+                        "Unused call specification 'A.CallTo(() => foo.Bar())'; did you forget to configure or assert the call?",

```

Comment:
It's not really a refactoring, since the functionality of the analyzer has also changed. I'll let it pass though :wink: